official/transformer/v2/transformer_main.py

```diff
@@ -121,6 +122,12 @@ def __init__(self, flags_obj):
   def train(self):
     """Trains the model."""
     params, flags_obj, is_train = self.params, self.flags_obj, True
+    # Sets config options.
+    keras_utils.set_config_v2(
```

Comment:
I think the transformer model supports TF 1. A simple fix would be that if TF1 is used, and either enable_xla==False or enable_grappler_layout_optimizer==True, then raise an error saying those flags are only supported with TF 2.

Reply:
I will add in the TF 1.0 config code.  I was not sure this works with TF 1.0.  I might have been thinking of the estimator version not being upgraded for TF 2.0.

Reply:
I refactored this entirely.  I believe I have thought it through and saved a lot of lines of code and made this easier to reuse.